packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1068,6 +1068,12 @@ export default class TradeStore extends BaseStore {
             form_name: 'default',
             trade_type_name: getContractTypesConfig()[this.contract_type]?.title,
             ...options,
+            ...(options.action === 'change_parameter_value' && options.duration_type
+                ? {
+                      duration_type:
+                          ANALYTICS_DURATIONS.find(value => value.startsWith(options.duration_type ?? '')) ?? '',
```

Comment:
If `options.duration_type` isn't provided, it will find and use the first type from  `ANALYTICS_DURATIONS` array here which is `ticks`. Is this the intended behavior?

Reply:
But there is a check above (line 1071), if `options.duration_type` exists or not

Reply:
Ok, if that's the case, there's no need to add ` ?? ''`. This won't ever get used?

Reply:
@akmal-deriv 
- If you mean `startsWith(options.duration_type ?? ''))`, `?? ''` is needed here because TS doesn't see that `options.duration_type` will not be undefined by the time this line is executed,
- If you mean the second  ?? '', it's needed here because we need to default to something in case `options.duration_type` is none of these: 't', 's', 'm', 'h', 'd' :)